pygsheets/cell.py

```diff
@@ -278,7 +296,7 @@ def get_json(self):
             nformat, pattern = self.format, ""
         return {"userEnteredFormat": {
                         "numberFormat": {
-                            "type": nformat.value,
```

Comment:
Good work, everything works like a charm except this change. As per what you said, your `nformat` is `None` which shouldn't happen.